srtcore/congctl.cpp

```diff
@@ -141,13 +142,13 @@ class LiveCC: public SrtCongestionControlBase
 
         // XXX NOTE: TEV_SEND is sent from CSndQueue::worker thread, which is
         // different to threads running any other events (TEV_CHECKTIMER and TEV_ACK).
-        // The m_iSndAvgPayloadSize field is however left unguarded because as
-        // 'int' type it's considered atomic, as well as there's no other modifier
+        // The m_zSndAvgPayloadSize field is however left unguarded because 
+        // 'int' type is considered atomic, as well as there's no other modifier
```

Comment:
`'int' type it's considered atomic`, - 'int' is definitely not atomic. Please adjust this comment.